src/components/cursor-controller.js

```diff
@@ -129,9 +131,10 @@ AFRAME.registerComponent("cursor-controller", {
       }
       cursor.object3D.position.copy(cursorPose.position).addScaledVector(cursorPose.direction, this.distance);
       // The cursor will always be oriented towards the player about its Y axis, so objects held by the cursor will rotate towards the player.
-      camera.object3D.getWorldPosition(cameraPos);
+      getLastWorldPosition(camera.object3D, cameraPos);
       cameraPos.y = cursor.object3D.position.y;
```

Comment:
I see, so theoretically we should be transforming it into cursor space before calling lookAt. I'll leave it for now since that'll just add time to a hotspot